app/controllers/room_bookings_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Room Bookings Controller
+class RoomBookingsController < ApplicationController
+  def index
+    schedule_id = params[:schedule_id]
+    @schedule = Schedule.find(params[:schedule_id])
+    @rooms = @schedule.rooms.where(is_active: true).where.not(building_code: 'ONLINE')
+    @tabs = TimeSlot.distinct.pluck(:day)
+    @active_tab = params[:active_tab] || @tabs[0]
+    @time_slots = TimeSlot.where(time_slots: { day: @active_tab })
```

Comment:
The times can appear out of order in the table. Fix this by adding:
```
@time_slots = TimeSlot.where(time_slots: {day: @active_tab}).to_a
@time_slots.sort_by!{ |ts| Time.parse(ts.start_time) }
```
which puts the times in temporal order.

Reply:
Good catch. Fixed.